setup.py

```diff
@@ -83,6 +83,7 @@
         "bumpversion>=0.5.3,<1",
         "wheel",
         "setuptools>=36.2.0",
+        "pluggy==0.7.1",
```

Comment:
I think if we add this as a standalone dependency to workaround the issue, we should put a comment above that line as an explainer. Something like:

>Pinning pluggy version to workaround incompatibility issues between certain `pytest` and `tox` versions.

Reply:
Done